jest-reporter.js

```diff
@@ -5,14 +5,14 @@ const path = require('path');
 const CI = require('./src/ci')
 
 const debug = (text) => {
-  if (process.env.BUILDKITE_ANALYTICS_DEBUG === "true") {
+  if (process.env.BUILDKITE_ANALYTICS_DEBUG_ENABLED === "true") {
```

Comment:
I have restored this name to make it consistent with the ruby gem and the docs.

I like the shorter name so I'll pop a ticket in our queue to change it here, in the ruby gem, and on the docs.